content-repo/extra-docs/releases/20.6.1.md

```diff
@@ -0,0 +1,599 @@
+
+DISCLAIMER: Please consult with your legal team before implementing this playbook.
+
+** Source: https://www.hhs.gov/hipaa/for-professionals/breach-notification/index.html
+
+
+---
+
+### New: Infocyte Pack v1.0.0
+#### Integrations
+##### Infocyte  
+Infocyte can pivot off incidents to automate triage, validate events with forensic data and enabling dynamic response actions against any or all host using both agentless or agented endpoint access.
+
+
+---
+
+### New: PCAP Analysis Pack v1.0.0
+#### Scripts
+##### PcapMinerV2  
+PcapMIner V2 allows to parse PCAP files by displaying the all of the relevant data within including ip addresses, ports, flows, specific protocol breakdown, searching by regex, decrypting encrypted  traffic and more.
+This automation takes about a minute to process 20,000 packets (which is approximately 10MB). If you want to mine large files you can either:
+a) Use the `pcap_filter` parameter to filter your PCAP file and thus make is smaller.
+b) Copy the automation and change the `default timeout` parameter to match your needs.
+
+
+---
+
+### New: Polygon Pack v1.0.0
+#### Integrations
+##### Group-IB TDS Polygon  
+TDS Polygon is a Malware Detonation & Research platform designed for deep dynamic analysis and enhanced indicators extraction. TDS Polygon analyzes submitted files and urls and extracts deep IOCs that appear when malicious code is triggered and executed. Polygon could be used either for application-level tasks (like smtp-based mail filtering) and analytical purposes (files/urls analysis for verdict, report and indicators).
+#### Playbooks
+##### Detonate File - Group-IB TDS Polygon  
+Detonate file using Group-IB TDS Polygon integration. This playbook returns relevant reports to the War Room and file reputations to the context data. The detonation supports the following file types: 7z, ace, ar, arj, bat, bz2, cab, chm, cmd, com, cpgz, cpl, csv, dat, doc, docm, docx, dot, dotm, dotx, eml, exe, gz, gzip, hta, htm, html, iqy, iso, jar, js, jse, lnk, lz, lzma, lzo,  lzh, mcl, mht, msg, msi, msp, odp, ods, odt, ots, ott, pdf, pif, potm, potx, pps, ppsm, ppsx, ppt, pptm, pptx, ps1, pub, py, pyc, r*, rar, reg, rtf, scr, settingcontent-ms, stc, svg, sxc, sxw, tar, taz,  .tb2, .tbz, .tbz2, tgz, tlz, txz, tzo, txt, url, uue, vbe, vbs, wsf, xar, xls, xlsb, xlsm, xlsx, xml, xz, z*, zip.
+##### Detonate URL - Group-IB TDS Polygon 
+Detonate URL using Group-IB TDS Polygon integration.
+
+
+---
+
+### AWS - EC2 Pack v1.1.1
+#### Scripts
+##### AwsEC2GetPublicSGRules
+- Added support for security groups with only one ingress rule.
+- Playbook to sync a list of IPs with an AWS Security Group. Moved from IPWhitelisting pack.
+
+---
+
+### Base Pack v1.0.10
+#### Scripts
+##### CommonServerPython  
+- Added support for the CVE indicator class.
+- Add safeget from python dict function.
+- Fixed an issue where the **argToList** function did not behave as expected. This fix breaks backward compatibility.
```

Comment:
What does this mean: "This fix breaks backward compatibility"?  @yaakovi I think you wrote this.

Reply:
@glicht before the change:
`argToList(1)` would output `1`
after the change
`argToList(1)` outputs `[1]`

Reply:
wasn't me, https://github.com/demisto/content/pull/7472